### ISSUE TITLE
Inconsistent performance with paper

### ISSUE THREAD

**Issue by ukaukaaaa** [NTUMIP402/Lung-Nodule-Detection#2]
Hi, thanks for this great code work, you guys did a good job. But I have a few questions.

After training with your released code, I wrote the missing jsongenerator code myself and create 5fold which contain two subsets in one cross. I train with the following command
`python main_detector_recon.py --model OSAF_YOLOv3 -b 16 --epochs 200 --save-dir OSAF_testcross4 --gpu '0' --n_test 1 --lr 0.001 --cross 4 `
in which the learning rate is 0.001 and epoch is 200. 

I got the following result which might be inconsistent with reported performance in your paper
![image](https://user-images.githubusercontent.com/42864050/184062493-3989547e-77cf-44d1-8e74-e77ac2afd02f.png)

![image](https://user-images.githubusercontent.com/42864050/184062565-3c4b7a6a-f7c6-40c7-a0a6-8e8c9a968c5d.png)

It shows a lot of false positive cases. I found that when in some of the cross, it predicts 400 boxes with ground truth 2 box for one patient while in other crosses, it predicts 10000 boxes with ground truth 1 box for one patient. It really confused me.

I wonder if there are any other training tricks that I missed.

**Comment by hongu0603** [comment]
I have the same question, did you solve it ?
